verilog/preprocessor/verilog_preprocess_test.cc

```diff
@@ -99,6 +101,13 @@ TEST(VerilogPreprocessTest, InvalidPreprocessorInputs) {
   }
 }
 
+#define EXPECT_PARSE_OK()                                                \
+  do {                                                                   \
```

Comment:
maybe try `SCOPED_TRACE` from googletest, which I believe shows test stacktraces useful for test helper functions.

Reply:
It looks like it will require one more line to add to each place I call this, so the advantage of only having one thing to call is diminishing..